mesonbuild/compilers/detect.py

```diff
@@ -648,6 +648,10 @@ def detect_cuda_compiler(env: 'Environment', for_machine: MachineChoice) -> Comp
         cpp_compiler = detect_cpp_compiler(env, for_machine)
         cls = CudaCompiler
         env.coredata.add_lang_args(cls.language, cls, for_machine, env)
+        for k in env.options:
+            if 'cuda' in k.name:
+                # To fix LDFLAGS issue
+                env.coredata.set_options({k: cls._to_host_flags_base(env.options[k], _Phase.LINKER)})
```

Comment:
Same here with `_Phase`, that leading underscore is a python convention that means `protected`.

Reply:
Changed accordingly.